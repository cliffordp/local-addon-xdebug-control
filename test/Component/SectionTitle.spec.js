const React = require( 'react' )
const Component = require( 'react' ).Component
const renderer = require( 'react-test-renderer' )

const context = {
	'React': React,
}

const SectionTitle = require( './../../src/Component/SectionTitle' )( context )

describe( '<SectionTitle/>', function () {
	const defaultProps = {
		text: 'foo',
	}

	it( 'renders correctly with props', function () {
		const component = renderer.create( <SectionTitle {...defaultProps}/> )
		let tree = component.toJSON()

		expect( tree ).toMatchSnapshot()
	} )
} )
