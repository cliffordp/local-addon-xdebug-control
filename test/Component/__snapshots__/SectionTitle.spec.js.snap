// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle/> renders correctly with props 1`] = `
<h4
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  foo
</h4>
`;
